hr_employee_calendar_planning/tests/test_hr_employee_calendar_planning.py

```diff
@@ -170,10 +170,11 @@ def test_calendar_planning(self):
 
         self.assertTrue(self.employee.resource_calendar_id.two_weeks_calendar)
 
-        # Calendar 1 has 20 lines + Calendar 2 has 6 lines that are duplicated
-        # in the odd and even week + even week label + odd week label
+        # Calendar 1 has 30 lines including lunch times
+        # Calendar 2 has 6 lines that are duplicated in the odd and even week
+        # + even week label + odd week label
         self.assertEqual(
-            len(self.employee.resource_calendar_id.attendance_ids), 20 + 6 * 2 + 2
+            len(self.employee.resource_calendar_id.attendance_ids), 30 + 6 * 2 + 2
```

Comment:
Why all these number changings? Can you please avoid change them? The test should be immutable as a check of the functionality is preserved (unless something involved with data initialization itself).

Reply:
Absolutely. The idea of a unitary test is to protect the technical evolution of the module. In this case the attendance_ids was changed because of the inclusion of the breaks... I do come back to you on Wednesday with concrete details of it as I did this work some time ago.

Reply:
I see that we are creating the calendar deterministically, so the number should be fixed. Maybe this version is adding calendar attendances entries automatically that changes this number, that you should remove or prevent to be created, or maybe you need to filter according the criteria to get the same number.

Reply:
@pedrobaeza did a pre-filtering of those attendances